source/_integrations/immich.markdown

```diff
@@ -0,0 +1,61 @@
+---
+title: Immich
+description: Instructions on how to integrate an immich user account into Home Assistant.
+ha_category:
+  - Sensor
```

Comment:
```suggestion
  - Multimedia
  - Sensor
```

Reply:
at the moment, there are no media platforms in the integration, so this should be updated, when these platroms get added

Reply:
But the integration itself still has to do with multimedia :)